src/envs/pybullet_env.py

```diff
@@ -251,8 +244,10 @@ def _detect_held_object(self) -> Optional[int]:
         one that is closest.
         """
         expected_finger_normals = {
-            self._pybullet_robot.left_finger_id: np.array([0., 1., 0.]),
-            self._pybullet_robot.right_finger_id: np.array([0., -1., 0.]),
+            self._pybullet_robot.left_finger_id: np.array([np.pi / 2., 1.,
```

Comment:
i don't understand what this change does, but somehow it works for both cover and blocks (which use two different ee_orientations). the previous thing didn't work for cover.

Reply:
sounds a little fluky. can we try to understand what's going on here before merging?

Reply:
sure will look into it tomorrow

Reply:
nvm looked into it now, tl;dr is you're right it's a bad mistake, i'll make this env-specific.

Reply:
resolved in latest commit